usaspending_api/api_contracts/contracts/v2/search/spending_by_category/awarding_agency.md

```diff
@@ -0,0 +1,166 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Spending By Awarding Agency [/api/v2/search/spending_by_category/awarding_agency/]
+
+This endpoint supports the advanced search page and allow for complex filtering for specific subsets of spending data.
+
+## POST
+
+This endpoint returns a list of the top results of Awarding Agencies sorted by the total amounts in descending order.
+
++ Request (application/json)
+    + Attributes (object)
+        + `filters` (required, FilterObject)
+            The filters to find with said category
+        + `limit`: 5 (optional, number)
+            The number of results to include per page
+        + `page`: 1 (optional, number)
+            The page of results to return based on the limit
+        + `subawards` (optional, boolean)
+            Determines whether Prime Awards or Sub Awards are searched
+    + Body
+        
+            {
+                "filters": { 
+                    "keywords": ["Filter is required"] 
+                }
+            }
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + `category`: `awarding_agency` (required, string)
```

Comment:
Typically the Attributes don't have example values and we include an example JSON body

Reply:
Followup-thought:

This value should always be `awarding_agency ` which means it is unnecessary.
Although, I assume it is present for interoperability between the original SpendingByCategory response.

Open to discussion if it is necessary

Reply:
Your assumption is correct. It doesn't need to be there, but not having there would make it differ from the current SpendingByCategory response. Removing it means adding logic that will eventually be redundant when we move away from a single SpendingByCategory response.